src/censored.jl

```diff
@@ -0,0 +1,409 @@
+"""
+    censored(d0::UnivariateDistribution, l::Union{Real,Missing}, u::Union{Real,Missing})
+
+A _censored distribution_ `d` of a distribution `d0` to the interval `[l, u]` has the
+probability density (mass) function:
+
+```math
+f(x; d_0, l, u) = \\begin{cases}
+    P_{Z \\sim d_0}(Z \\le l), & x = l \\\\
+    f_{d_0}(x),              & l < x < u \\\\
+    P_{Z \\sim d_0}(Z \\ge u), & x = u \\\\
+  \\end{cases}, \\quad x \\in [l, u]
+```
+where ``f_{d_0}(x)`` is the probability density (mass) function of ``d_0``.
```

Comment:
Usually I use the convention to write `f(x)` for the value of a function `f` at `x`, hence I'd use
```suggestion
where ``f_{d_0}`` is the probability density (mass) function of ``d_0``.
```
here.